network-api/networkapi/wagtailpages/factory/bannered_campaign_page.py

```diff
@@ -45,9 +48,23 @@ def generate(seed):
     reseed(seed)
 
     print('Generating Bannered Campaign Pages under namespace')
-    BanneredCampaignPageFactory.create(
-        parent=bannered_campaign_namespace,
-        title="Test Bannered Campaign"
-    )
+    title = 'Initial test Bannered Campaign with fixed title'
```

Comment:
We should update this line as well: https://github.com/mozilla/foundation.mozilla.org/blob/master/cypress/integration/endpoint-tests.js#L118

Reply:
I wish Percy showed us a warning sign instead of ✅ 

https://percy.io/Mozilla-Foundation/foundation.mozilla.org/builds/3347705?utm_campaign=Mozilla-Foundation&utm_content=foundation.mozilla.org&utm_source=github_status_public

<img width="615" alt="image" src="https://user-images.githubusercontent.com/2896608/70099323-5b323d00-15e3-11ea-8230-c2a652034325.png">

Reply:
man, yeah... =(

test updated.